packages/schema/bind/src/bindings/assemblyscript/wasm/templates/msgpack/uint32-ts.mustache

```diff
@@ -0,0 +1,20 @@
+{{> serialization_imports}}
+
+export class UInt32 {
```

Comment:
Why is this being added directly to the template, and not re-exported from the `@polywrap/wasm-as` package?